backend/django/api/views.py

```
@@ -1,92 +1,217 @@
+            "tasks",
+            queryset=Task.objects.only(
+                "id", "title", "done", "project_id", "created_at"
+            ),
+        )
+    ).order_by("-created_at")
+
     serializer_class = ProjectSerializer
     permission_classes = [IsAuthenticated]
     pagination_class = PageNumberPagination
 
+    def list(self, request, *args, **kwargs):
```

Ruff (0.14.1)

Unused method argument: `request`

(ARG002)